### DISCUSSION TITLE
138

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#71] [General]
# 潮流周刊第138期 - 下班蓝天

封面图来源于上周某天下班时候在路上随手拍的一张照片，很普通的蓝天夜晚，不过心情太好，看啥都很舒服的。 

https://weekly.tw93.fun/posts/138-%E4%B8%8B%E7%8F%AD%E8%93%9D%E5%A4%A9

**Comment by HaofindU** [comment]
发布日期7.24？

**Reply by tw93** [reply]
懒，提前一天发布，时间还是周一

**Reply by HaofindU** [reply]
haha……从你的分享里面看到很多有趣的东西，谢谢哥

**Comment by fantingsheng** [comment]
第一次进来，留个言吧

**Reply by tw93** [reply]
哈哈 随便说 随便聊

**Comment by tangmengqiu** [comment]
图片和文章排版都很舒服

**Comment by ChengWeiDong** [comment]
原来老哥在EFC哈哈哈哈

**Reply by tw93** [reply]
哈哈哈哈哈🤣
